src/aics-image-viewer/components/ChannelsWidget/index.tsx

```diff
@@ -31,16 +32,13 @@ interface ChannelSettings {
   opacity: number;
   color: [number, number, number];
   dataReady: boolean;
-  controlPoints: [];
+  controlPoints: {
+    color: string;
+    opacity: number;
+    x: number;
+  }[];
```

Comment:
a type definition for this might be in order some time.  it would be interesting to see if we could define it all the way down in volume-viewer and just reuse it up here.

Reply:
This is already basically duplicated from `App/types.ts`, so it would definitely useful to have a single source of truth for this interface.

Reply:
I actually meant just the control point itself (x, color, opacity) living down in volume-viewer.  But I agree with what you said too.. duplicate type defs are worse than duplicated code